### ISSUE TITLE
How to get working in browser

### ISSUE THREAD

**Issue by MrMoronIV** [commenthol/date-holidays#49]
Hi,

I'm at loss, how would one get this script to work in a regular browser?
Do I really need node to get this script going or can I use it at the frontend as well?
It's suggested it is possible but how?

Aren't there some ready to grab script files I can inlcude on my page, include some locales and work with it?

**Comment by commenthol** [comment]
Hi, please clone the repo, check the folder examples/browser and follow the README. Feel free to inspect the code there as well. This should give you a sample on how to integrate in the browser. Please bear in mind that the full lib is not lightweight in size. 
On the other hand you may want to bundle with webpack `npm run webpack` may give to the script files you like to grab...
You may consider selecting only the countries you require. Please see the README of this project on how to achieve this.

**Comment by commenthol** [comment]
Closing...

**Comment by FrankW76** [comment]
I am also at a complete loss of how to run this in a browser. I have it running in nodejs but I have no clue how to get this working in a browser.

I tried browerify. But it gives errors. I tried babelify but it gave other errors. I tried other things, but all giving errors.


Looked for the README file but cannot find it. I can get the example to run but I still have no clue as index.js seems to contain minified code and things like that.

I just want to add a script tag to my html file and then be able to things like 'new Holiday'?
